projects/angular/src/forms/styles/_toggles.clarity.scss

```
@@ -55,56 +61,65 @@
         position: absolute;
         display: inline-block;
         content: '';
-        border: tokens.$cds-alias-object-border-width-200 solid toggle-variables.$clr-toggle-bg-color-off;
-        border-radius: tokens.$cds-alias-object-border-radius-200;
-        height: toggle-variables.$clr-toggle-switch-base-height;
-        width: toggle-variables.$clr-toggle-switch-base-width;
-        background-color: toggle-variables.$clr-toggle-bg-color-off;
+        border-radius: tokens.$cds-alias-object-border-radius-300;
+        border: tokens.$cds-alias-object-border-width-400 solid toggle-variables.$clr-toggle-handle-border-color;
+        background-color: toggle-variables.$clr-toggle-handle-bg-color;
+        left: 0;
+        right: calc(-1 * toggle-variables.$clr-toggle-switch-base-width);
+
+        @include mixins.equilateral(toggle-variables.$clr-toggle-dot-diameter);
         @include clr-toggle-transition-styles;
-        transition-property: border-color, background-color;
+        transition-property: right, left, border;
       }
 
       //Toggle Switch
       &::after {
         position: absolute;
-        display: inline-block;
         content: '';
-        border: tokens.$cds-alias-object-border-width-200 solid toggle-variables.$clr-toggle-handle-border-color;
-        border-radius: tokens.$cds-alias-object-border-radius-300;
-        background-color: toggle-variables.$clr-toggle-handle-bg-color;
-        left: 0;
-        right: calc(-1 * toggle-variables.$clr-toggle-switch-base-width);
-
-        @include mixins.equilateral(toggle-variables.$clr-toggle-dot-diameter);
+        visibility: hidden;
         @include clr-toggle-transition-styles;
-        transition-property: right, left;
+        transition-property: visibility, right, left, border;
+        left: #{tokens.$cds-global-space-3};
+        right: 0;
+        height: calc(#{$mark-half-size} + #{tokens.$cds-global-space-1});
+        width: forms-variables.$clr-forms-checkbox-mark-size;
+        border-left: forms-variables.$clr-forms-checkbox-mark-border-size solid
+          toggle-variables.$clr-toggle-bg-color-off;
+        border-bottom: forms-variables.$clr-forms-checkbox-mark-border-size solid
+          toggle-variables.$clr-toggle-bg-color-off;
+        transform: translate(0, $mark-half-size) rotate(-45deg);
```

While testing I duscovered that the `right` calculation is not needed.
You can freely remove it.

```suggestion
        transition-property: visibility, left, border;
        left: #{tokens.$cds-global-space-3};
        height: calc(#{$mark-half-size} + #{tokens.$cds-global-space-1});
        width: forms-variables.$clr-forms-checkbox-mark-size;
        border-left: forms-variables.$clr-forms-checkbox-mark-border-size solid
          toggle-variables.$clr-toggle-bg-color-off;
        border-bottom: forms-variables.$clr-forms-checkbox-mark-border-size solid
          toggle-variables.$clr-toggle-bg-color-off;
        transform: translate(0, $mark-half-size) rotate(-45deg);
```